packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -216,7 +216,8 @@ export class SummaryTreeBuilder implements ISummaryTreeWithStats {
 	 * Adds an {@link @fluidframework/driver-definitions#ISummaryHandle} that references a subtree, blob, or attachment in a previous summary.
 	 *
 	 * @remarks
-	 * There are special limitations to both the key and handle parameters: We use encodeURIComponent and decodeURIComponent to encode and decode the key and handle parameters after they are added to the summary tree. This means that the key and handle parameters must be valid URI components. If they are not, the encoding and decoding will fail and the summary will not be generated correctly.
+	 * Previously we had had special limitations to both the key and handle parameters since we used encodeURIComponent and decodeURIComponent to encode and decode the key and handle parameters after they are added to the summary tree.
```

Comment:
Why do we need to mention this? It adds no value and may just add confusion. I would suggest removing this comment altogether.